core/src/test/scala/org/apache/spark/crypto/CryptoCodecSuite.scala

```diff
@@ -0,0 +1,201 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.crypto
+
+import java.io._
+import java.security.SecureRandom
+import java.util
+import java.util.Random
+
+import com.intel.chimera._
+import com.intel.chimera.codec._
+import org.apache.spark.SparkConf
+
+import org.apache.spark.SparkFunSuite
+import org.apache.spark.crypto.CommonConfigurationKeys._
+
+private[spark] class CryptoCodecSuite extends SparkFunSuite {
```

Comment:
What Spark code is this test testing? It seems to only be testing the library itself.